dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpGenotypeTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.models;
+
+import org.junit.Test;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+
+public class SubSnpGenotypeTest {
+
+    @Test
+    public void testSubSnpGenotypeFields() {
+        List<String> genotypeList = Arrays.stream("A,A, G".split(",")).map(String::trim)
```

Comment:
This test will be more reliable like this: List<String> genotypeList = Array.asList("A", "A", "G");
Using the same code here and in the SubSnpGenotype constructor (`Arrays.stream().map(String::trim).collect(... `) is not safe because you could be having the same bug in both places. (See https://github.com/mockito/mockito/wiki/How-to-write-good-tests#avoid-coding-a-tautology)